test/test_address_nl.rb

```diff
@@ -23,7 +23,7 @@ def test_province
   end
 
   def test_zip_code
-    assert_match(/\A[1-9]\d{3} [A-Z]{2}\z/, @tester.zip_code)
+    assert_match(/\A[1-9]\d{3} ?(?!sa|sd|ss)[A-Z]{2}\z/, @tester.zip_code)
```

Comment:
This matcher will assert with '1234SA', '1234 SA' 
See https://rubular.com/r/wgt0m3VApIWJrr

maybe)
```suggestion
    assert_match(/\A[1-9]\d{3} [A-RT-Z][A-Z]\z/, @tester.zip_code)
```
will be fine.